roadmap/roadmap.md

```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
 
 ## Functionality
 
-- :large_orange_diamond: Provide next-generation spectroscopic reduction, analysis, and visualization tools usable by individual researchers and larger surveys.
+The core product of the Astropy Project is high quality astronomical software, including both the core library and Astropy Affiliated packages. This category encompasses goals having to do with this core Project functionality.
+
+- :green_circle: Provide next-generation spectroscopic analysis tools usable by individual researchers and larger surveys.
+
+- :green_circle: Add quantity support for non-NumPy arrays.
+
+- :large_orange_diamond: Provide next-generation spectroscopic reduction tools usable by individual researchers and larger surveys.
+
+
+## Infrastructure, Documentation
+
+Underlying all Astropy packages is critical infrastructure that ensures high quality, well-documented software is released. This category encompasses goals having to do with such infrastructure and documentation.
+
+
+- :green_circle: Implement robust performance benchmark reporting.
+
+- :green_circle: Revive dashboards for developers (CI status etc) and community (PR trends etc).
+
+- :large_orange_diamond: Improve documentation for infrastructure setup, as well as the developer documentation.

```

Comment:
Actually, @taldcroft already improved developer docs in https://github.com/astropy/astropy/pull/16561 . Does that mean we can take developer docs off this roadmap? Or change status for that one?

Reply:
Let's change to green and leave on.